docs/gencontroller_config.md

```diff
@@ -0,0 +1,55 @@
+# GenController Configuration
+
+## NumTeams
+
+*int64*
+
+The number of teams to be created.
+
+## NumChannels
+
+*int64*
+
+The number of channels to be created.
+
+## NumPosts
+
+*int64*
+
+The number of posts to be created.
+
+## NumReactions
+
+*int64*
+
+The number of reactions to be created.
+
+## PercentReplies
+
+*float64*
+
+The percentage of replies (over the total number of posts) to be created.
+
+## PercentPublicChannels
+
+*float64*
+
+The percentage of public channels to be created.
+
+## PercentPrivateChannels
+
+*float64*
+
+The percentage of private channels to be created.
+
+## PercentDirectChannels
+
+*float64*
+
+The percentage of direct channels to be created.
+
+## PercentGroupChannels
+
+*float64*
+
+The percentage of group channels to be created.
```

Comment:
We should also mention that the sum of these should be equal to 1.

Reply:
Good point. I'll add a note.